src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+
+</Column>
+</Row>
+
+<Caption>
+  A visual spec using layer set tokens (left) versus contextual tokens (right).
+</Caption>
+
+<!---
+### Token list
+
+| Contextual tokens         | Layer set 01                 | Layer set 02                 | Layer set 03                 |
+| ------------------------- | ---------------------------- | ---------------------------- | ---------------------------- |
+| `$layer`                  | `$layer-01`                  | `$layer-02`                  | `$layer-03`                  |
+| `$layer-active`           | `$layer-active-01`           | `$layer-active-02`           | `$layer-active-03`           |
+| `$layer-hover`            | `$layer-hover--01`           | `$layer-hover-02`            | `$layer-hover-03`            |
+| `$layer-selected`         | `$layer-selected-01`         | `$layer-selected-02`         | `$layer-selected-03`         |
+| `$layer-selected-hover`   | `$layer-selected-hover-01`   | `$layer-selected-hover-02`   | `$layer-selected-hover-03`   |
+| `$layer-accent`           | `$layer-acccent-01`          | `$layer-accent-02`           | `$layer-accent-03`           |
+| `$layer-accent-active`    | `$layer-acccent-active-01`   | `$layer-accent-active-02`    | `$layer-accent-active-03`    |
+| `$layer-accent-hover`     | `$layer-acccent-hover-01`    | `$layer-accent-hover-02`     | `$layer-accent-hover-03`     |
+| `$field`                  | `$field-01`                  | `$field-02`                  | `$field-03`                  |
+| `$field-hover`            | `$field-hover-01`            | `$field-hover-02`            | `$field-hover-03`            |
+| `$border-subtle`          | `$border-subtle-01`          | `$border-subtle-02`          | `$border-subtle-03`          |
+| `$border-subtle-selected` | `$border-subtle-selected-01` | `$border-subtle-selected-02` | `$border-subtle-selected-03` |
+| `$border-strong`          | `$border-strong-01`          | `$border-strong-02`          | `$border-strong-03`          |
+--->
+
+<br />
+
+### Layer set tokens
+
+Layer tokens are sets of explicit tokens used to manually map the layering model
+onto components. They are part of the theme package and come in predefined sets
+that pair UI and interaction colors together to accurately match the layering
+model in each theme.
+
+There are four sets of color layering within a theme: base layer, layer 01,
+layer 02, and layer 03. After each layer of color added you move up to the next
+layering level and use the next set of layering tokens. For example, on top of
+`$layer-01` you would place `$layer-02`. These layer set tokens are identified
+by a number suffix (-01, -02, -03) attached to the core token name, such as
+`$field-03`. The number indicates which set the token is a part of. _Note:
+previously in v10 many color tokens had numeral suffixes, now in v11 only
+layering tokens will have this distinction._
+
+Not all layer set tokens use `$layer` as its token name. Token sets also include
```

Comment:
Seems like we're saying "layer set tokens" and then "token sets". Are they two separate things or should they be the same term?